From a3b4570aeb8582e0b0f2053dd78fd65befc1cad5 Mon Sep 17 00:00:00 2001
From: "hainan.wang" <hainan.wang@unisoc.com>
Date: Fri, 16 Sep 2022 18:29:18 +0800
Subject: [PATCH] Bug #2011601 delete-neverallow-proc_kmsg_t

[root cause  ]: x
[changes     ]:x
[side effects]:xx
[self test   ]:x
[download normally]:x
[power on/off normally]:x
[do common repository/branch inspection]:x
[is there dependence]:x
[confirm dependent commit]:x
[board]:x
[test case]:x
[reviewers]:x

commit_template_version:v1
---
 policy/modules/kernel/kernel.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/policy/modules/kernel/kernel.te b/policy/modules/kernel/kernel.te
index 7b81c7327..b51035169 100644
--- a/policy/modules/kernel/kernel.te
+++ b/policy/modules/kernel/kernel.te
@@ -84,7 +84,6 @@ genfscon proc /fs/openafs gen_context(system_u:object_r:proc_afs_t,s0)
 # kernel message interface
 type proc_kmsg_t, proc_type;
 genfscon proc /kmsg gen_context(system_u:object_r:proc_kmsg_t,mls_systemhigh)
-neverallow ~{ can_receive_kernel_messages kern_unconfined } proc_kmsg_t:file read;
 
 # /proc kcore: inaccessible
 type proc_kcore_t, proc_type;
-- 
2.17.1

